### ISSUE TITLE
show some nice screenshots

### ISSUE THREAD

**Issue by DreadKnight** [drawpile/Drawpile#1404]
Website and GitHub README should have some screenshots of the app in action, ideally with some nice enough artwork on the canvas itself.
I've barely found a single boring screenshot in Kubuntu's Discover, after searching for the app to see a screenie because website and GitHub failed me big time. This is bad at attracting new people and marketing. See my open source game repo here -> https://github.com/FreezingMoon/AncientBeast it shows a nice screenie first thing in the repo's README. Image not having that, no idea, boring, skip/close asap. I see this is a decade old project at least, even if there was no app, a mock-up would be doing wonders in attracting users and developers. You get the idea.

**Comment by askmeaboutlo0m** [comment]
Yup, having screenshots with a pile of art would be nice, I just gotta go around the artist groups I'm in and ask them to add to a picture for it and agree to release it into Creative Commons so that we can actually use it.

For what it's worth, Drawpile mostly spreads because existing users of it rope their friends into drawing with them, not because someone stumbles across the program on their own. There's also a bit of work to do with e.g. #1207 to make it a bit more palatable to new users that don't have friends to help them out setting the thing up.
